microdnf.spec

```diff
@@ -21,7 +21,7 @@ BuildRequires:  help2man
 Requires:       libdnf%{?_isa} >= %{libdnf_version}
 %if 0%{?rhel} > 8 || 0%{?fedora}
 # Ensure DNF package manager configuration skeleton is installed
-Requires:       dnf-data
+Requires:       libdnf5
```

Comment:
I guess the require requires to add a version.
`Requires:       libdnf5 >= <version of libdnf5 that contains dnf.conf>`
I guess that the version is `5.0.12`

Reply:
The above solution will help but in case that you will install package from multiple version of fedora it will also fail. I suggest to use `Requires:       /etc/dnf/dnf.conf`.

Reply:
Ah makes sense, I've changed it to /etc/dnf/dnf.conf.